src/webportal/src/app/job/job-view/fabric/job-detail/conn.js

```diff
@@ -100,6 +100,27 @@ export async function fetchSshInfo() {
   }
 }
 
+export function getTensorBoardUrl(jobInfo, rawJobConfig) {
+  let port = null;
+  let ip = null;
+  if (rawJobConfig.hasOwnProperty('extras') && rawJobConfig.extras.hasOwnProperty('tensorBoardStr')) {
+    const tensorBoardStr = rawJobConfig.extras.tensorBoardStr;
+    const obj = jobInfo.taskRoles;
+    Object.keys(obj).forEach(function(key) {
+      if (key === 'TensorBoard_' + tensorBoardStr) {
+        if (obj[key].taskStatuses['0'].taskState == 'RUNNING') {
+          port = obj[key].taskStatuses['0'].containerPorts['tensorBoardPort_' + tensorBoardStr];
```

Comment:
`taskStatuses` is an array, just use `taskStatuses[0]` instead of `'0'`

Reply:
Got it. Thanks.